llama.h

```diff
@@ -235,6 +235,7 @@ extern "C" {
         uint32_t seed;              // RNG seed, -1 for random
         uint32_t n_ctx;             // text context, 0 = from model
         uint32_t n_batch;           // prompt processing maximum batch size
+        uint32_t n_parallel;        // number of parallel sequences
```

Comment:
Could we avoid adding `n_parallel` and use `n_ctx` as usual? Could be clarified in the comment that for recurrent models, `n_ctx` is the number of states / sequences maintained